asana-helpers-markdown.js

```diff
@@ -182,6 +258,7 @@
     for (var i = 1; i <= 5; i++) {
       setTimeout(showComments, i * 100);
     }
+    animateButton(ev.srcElement);
```

Comment:
It might be more robust to store the IDs of these elements rather than relying on the click handler, but both work.